tickets/tickets_bot.go

```diff
@@ -210,8 +208,28 @@ func closeTicket(gs *dstate.GuildSet, currentTicket *Ticket, ticketCS *dstate.Ch
 		closingTicketsLock.Unlock()
 	}()
 
+	closingMsg := "Closing ticket."
+
+	// We only need to build up a more detailed closing msg.
+	// if we're creating logs.
```

Comment:
```suggestion
	// We only need to generate a more detailed closing message if we're creating logs.
```
the full stop is misplaced as is